utils/utils.py

```diff
@@ -5,18 +5,24 @@
 import time
 
 
+DEFAULT_CONFIG_FILE = "config.json"
+
+
 class PublicAPIClient:
-    def __init__(self, subdomain):
-        self.subdomain = subdomain
+    def __init__(self, config_file=DEFAULT_CONFIG_FILE):
+        self.config_file = config_file
         self.config_path = os.path.realpath(
-            os.path.join(os.path.dirname(__file__), "..", "config.json")
+            os.path.join(os.path.dirname(__file__), "..", self.config_file)
         )
         self._load_config()
+        self.subdomain = self.config["subdomain"]
+        self.domain = self.config.get("domain", "instructuremedia.com")
+        self.scheme = self.config.get("scheme", "https")
```

Comment:
I think we should document it somewhere, or add it to the generated config json with the default values or something. Currently only ppl who dig into the codebase can find them.

Reply:
True that. I'll add it to the README as an other "advanced" topic. I wouldn't add it to the generated config json, since it's only used by us.

Reply:
Besides the request above, the change looks good to me ( tested on staging and locally with test/main.py )

Reply:
Thanks for testing. I updated the README.